### PR TITLE
Update routing.rst

### PR DESCRIPTION
**Pull request by atailouloute**
Add missing uses

### CODE REVIEW

File: components/routing.rst [hunk 1]
```diff
@@ -206,6 +207,9 @@ to find a route that fits the given request you can also build a URL from
 a certain route::
 
     use Symfony\Component\Routing\Generator\UrlGenerator;
+    use Symfony\Component\Routing\RouteCollection;
+    use Symfony\Component\Routing\RequestContext;
```
Comment (HeahDude) [inline components/routing.rst]: Can you please keep the alpha order?

**Review by HeahDude — COMMENTED** [review]
Small comment, but 👍

File: components/routing.rst (hunk 1 above)
Comment (atailouloute) [inline components/routing.rst]: Done

**Review by javiereguiluz — APPROVED** [review]
:+1:

**Comment by xabbuh** [comment]
Thank you for the many good catches and improvements @atailouloute! To ease the merging of all the small fixes I will combine your PRs in a single PR (see #7737) and merge that one instead.

**Pull request closed without merging**